raiden/api/rest.py

```diff
@@ -154,7 +155,10 @@ def handle_request_parsing_error(err):
 
 
 def endpoint_not_found(e):
-    return api_error('invalid endpoint', HTTPStatus.NOT_FOUND)
+    errors = ['invalid endpoint']
+    if type(e) is InvalidEndpoint:
```

Comment:
Is that intentionally a check for only that specific type? Otherwise, I'd use `isinstance()`.